apps/condo/domains/common/utils/sberCloudFileAdapter.js

```diff
@@ -229,25 +231,30 @@ const obsRouterHandler = ({ keystone }) => {
                 returnFields += `, ${propertyQuery}`
             }
 
-            const item = await getItem({
-                keystone,
-                listKey,
-                itemId,
-                context,
-                returnFields,
-            })
+            hasAccessToReadFile = false
 
-            // item accessible
-            hasAccessToReadFile = !isNil(item)
-
-            // check property access case
-            if (hasAccessToReadFile && isString(propertyQuery) && !isNil(propertyValue)) {
-                const propertyPath = propertyQuery
-                    .replaceAll('}', '') // remove close brackets of sub props querying
-                    .split('{') // work with each path parts separately
-                    .map(path => path.trim()) // since gql allow to have spaces in querying - let's remove them
-                    .join('.') // join by . for lodash get utility
-                hasAccessToReadFile = get(item, propertyPath) == propertyValue
+            try {
+                const item = await getItem({
```

Comment:
it looks like you don't need to check access here. Use `getByID` util here

Reply:
No, I think this code was supposed to check access by design

Reply:
Gonna look into it :/

Reply:
We also have execute gql by user. It looks like we need to use it here?!